nix/libvirtd.nix

```diff
@@ -53,6 +53,32 @@ in
       '';
     };
 
+    deployment.libvirtd.template = mkOption {
```

Comment:
I guess I'm confused what's the point of having this as a nixops option ?
What if someone changes this to:
```
deployment.libvirtd.template = ''
<other template that doesn't use the same variables>
''
```
That'll basically make it inconsistent with what the python code expects